src/components/mdTable/mdTablePagination.vue

```diff
@@ -48,7 +48,7 @@
         subTotal: 0,
         currentSize: parseInt(this.mdSize, 10),
         currentPage: parseInt(this.mdPage, 10),
-        totalItems: !isNaN(this.mdTotal) && Number.MAX_SAFE_INTEGER
+        totalItems: isNaN(this.mdTotal)?Number.MAX_SAFE_INTEGER:parseInt(this.mdTotal, 10)
```

Comment:
The user can still pass a String to the `mdTotal`. The `totalItems` should be used only to disable the 'next' button. The fix is inside the line 9. Just change this `totalItems` to `mdTotal` on the template.